src/caselawclient/identifier_resolution.py

```diff
@@ -21,8 +21,10 @@ class IdentifierResolutions(list["IdentifierResolution"]):
     def from_marklogic_output(table: list[str]) -> "IdentifierResolutions":
         return IdentifierResolutions(list(IdentifierResolution.from_marklogic_output(row) for row in table))
 
-    def published(self) -> "IdentifierResolutions":
+    def published(self, only_published: bool = True) -> "IdentifierResolutions":
```

Comment:
Essentially I think `x.published(only_published=only_published)` is cleaner than

```
if only_published:
   new_x = x.published()
else:
   new_x = x
```

Reply:
I'm confused by what this is trying to achieve; isn't `.published` _always_ `only_published = True`?